adapters/repos/db/shard_async_replication.go

```diff
@@ -37,7 +39,104 @@ import (
 	enterrors "github.com/weaviate/weaviate/entities/errors"
 )
 
-const propagationLimitPerHashbeatIteration = 100_000
+const maxHashtreeHeight = 20
+
+const (
+	defaultHashtreeHeight              = 16
+	defaultFrequency                   = 5 * time.Second
+	defaultFrequencyWhilePropagating   = 10 * time.Millisecond
+	defaultAliveNodesCheckingFrequency = 1 * time.Second
+	defaultLoggingFrequency            = 3 * time.Second
+	defaultDiffPerNodeTimeout          = 10 * time.Second
+	defaultPropagationTimeout          = 30 * time.Second
+	defaultPropagationLimit            = 1_000_000
```

Comment:
Do you think maybe this is a bit high for the default?

Reply:
ok, reducing it back to 100_000